RELEASE/scripts/autoscend/quests/optional.ash

```diff
@@ -998,3 +998,24 @@ boolean LX_NemesisQuest()
 	if (LX_guildUnlock() || LX_acquireEpicWeapon()) { return true; }
 	return false;
 }
+
+void houseUpgrade()
+{
+	//function for upgrading your dwelling.
+	if(isActuallyEd() || my_class() == $class[Vampyre])
```

Comment:
Im pretty sure that you cant access your house and stuff in nuclear autumn too.

Reply:
thank you. fixed